src/utils.ts

```diff
@@ -147,3 +150,11 @@ export function normalizeWindowsImportId(id: string) {
   }
   return pathToFileURL(id);
 }
+
+export function isFipsMode() {
```

Comment:
we don't need to export
```suggestion
function isFipsMode() {
```